css/modal.css

```diff
@@ -0,0 +1,172 @@
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+  width: 90%;
+  height: calc(100vh - 10%);
+  position: fixed;
+  top: 0;
+  left: 5%;
+  bottom: 0;
+  margin: 10% auto 10%;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+  z-index: 2;
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: 1;
+  backdrop-filter: blur(15px);
+}
+
+.modal .description_wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 30px;
+}
+
+.project-image-container {
+  border-radius: 8px;
+  width: 100%;
+  height: 220px;
+  background-position: center;
+  background-size: 90%;
+  background-repeat: no-repeat;
+  overflow: hidden;
+}
+
+.dsktp {
+  display: none;
+}
+
+.mbl {
+  display: block;
+}
+
+.description_wrapper p.description {
+  color: #344563;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 32px;
+}
+
+.description_wrapper .tags-button_wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding-bottom: 1rem;
+}
+
+.live-source-container {
+  display: flex;
+  gap: 10px;
+}
+
+.live-source-container a {
+  border: 1px solid #6070ff;
+  text-decoration: none;
+  border-radius: 8px;
+  display: flex;
+  flex-wrap: nowrap;
+  padding: 5px 8px;
+  align-items: center;
+  gap: 10px;
+}
```

Comment:
- [ ] You're almost there! kindly ensure that your modal contains all the children elements as suggested earlier. The two `links` are displayed outside the modal container for the **`mobile version`**. 👍 

## Your Design
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/216387532-d3020c96-3b06-49d1-960b-ed7d429c96af.png)